python3.8/rockcraft.yaml

```diff
@@ -0,0 +1,39 @@
+name: python
+version: '3.8'
+summary: Python 3.8
+description: |
+    Python is an interpreted, interactive, object-oriented, open-source
+    programming language.
+license: GPL-3.0
+
+base: bare
+build-base: ubuntu@20.04
+
+run-user: _daemon_
+
+platforms:
+    amd64:
+
+services:
+    python3:
+        startup: disabled
+        override: replace
+        command: python3
+        on-success: ignore
+        on-failure: ignore
```

Comment:
Why ignore?

Reply:
I think this would vary per use-case. Some may ask for restarting the service, some shutdown and some may choose to not do anything. I left it as `ignore` to sort of accommodate all of them as if you need to restart or shutdown, you can do so from the console anyway.

But let me know if you think this should be `restart` or `shutdown`.

Reply:
I think it should be shutdown on both. If you only have one service, that's the standard expectation for a container